biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/ABITracerTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ *                    BioJava development code
+ *
+ * This code may be freely distributed and modified under the
+ * terms of the GNU Lesser General Public Licence.  This should
+ * be distributed with the code.  If you do not have a copy,
+ * see:
+ *
+ *      http://www.gnu.org/copyleft/lesser.html
+ *
+ * Copyright for this code is held jointly by the individual
+ * authors.  These should be listed in @author doc comments.
+ *
+ * For more information on the BioJava project and its aims,
+ * or to join the biojava-l mailing list, visit the home page
+ * at:
+ *
+ *      http://www.biojava.org/
+ *
+ */
+
+package org.biojava.nbio.core.sequence.io;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.net.URL;
+import org.junit.*;
+
+/**
+ * Test file 3730.ab1 is from https://github.com/biopython/biopython/blob/master/Tests/Abi/3730.ab1
+ * The test data for comparing the results from ABITracer for file 3730.ab1 is from https://github.com/biopython/biopython/blob/master/Tests/Abi/test_data
+ */
+public class ABITracerTest {
+
+    private String filePath = System.getProperty("user.dir") + "/src/test/resources/3730.ab1";
```

Comment:
This is not going to work when running as a jar. See how to do it here: https://stackoverflow.com/questions/3891375/how-to-read-a-text-file-resource-into-java-unit-test?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
I have a question regarding this. 
Is it possible to add the dependency for org.apache.commons.io.IOUtils? 
Its function IOUtils can be used to safely convert the InputStream to a byte[] and this can then be used for the Constructor ABITracer (byte[] ABIFileData).